pkg/client/attach.go

```diff
@@ -313,6 +343,9 @@ func (c *ConmonClient) readStdio(
 
 	case err = <-stdinDone:
 		c.logger.WithError(err).Trace("Received possible error on input channel")
+
+		c.tryCloseAttachReaderForID(id)
```

Comment:
hm we send on stdinDone in the CopyDetatchable goroutine. do we need to save the reader if we can close when CopyDetatchable exits?

Reply:
Hm I think we do not close at all in this case, will remove the code path.